packages/node/package.json

```diff
@@ -6,27 +6,37 @@
   "types": "dist/src/index.d.ts",
   "license": "MIT",
   "scripts": {
-    "build": "rollup -c",
-    "test": "jest --setupFiles dotenv-extended/config --runInBand --bail --forceExit",
+    "build": "tsc -p . && rollup -c",
+    "test": "yarn run test:no-ganache && yarn run test:with-ganache",
+    "test:with-ganache": "./scripts/test.sh",
+    "test:no-ganache": "jest --testPathIgnorePatterns=test/machine/integration --setupFiles dotenv-extended/config --runInBand --bail --forceExit",
```

Comment:
maybe `test:machine` and `test:rpc`?

Reply:
actually, the machine unit tests are run under `no-ganache` now, which I think is the right thing to do

Reply:
ok